prow/scripts/lib/gardener/gcp.sh

```diff
@@ -137,15 +137,12 @@ gardener::install_kyma() {
 }
 
 gardener::deploy_kyma() {
-    log::info "Deploying Kyma"
-
-    (
-    set -x
```

Comment:
can we either restore `set -x` or add the additional information given by `set -x` to the log::info